src/cascadia/TerminalApp/WindowsPackageManagerFactory.h

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
Agreed, which is why the nuget package is soon going to include a DLL that handles this part (exposes the activation factory so that one can just use WinRT language projections as normal).